src/app/api/users/users.controller.ts

```diff
@@ -8,14 +8,13 @@ export const getUsers = async (req: NextRequest) => {
   const user = await authenticate(req)
   const usersService = new UsersService(user)
   const keyword = req.nextUrl.searchParams.get('search')
-
   const rawLimit = req.nextUrl.searchParams.get('limit')
+  const userType = req.nextUrl.searchParams.get('userType') || undefined
```

Comment:
isnt it undefined if not found, already?

Reply:
If not sent, the `searchparams.get('userType')` will be null and not undefined. Since this is an optional parameter, i think we need to change it to undefined it we receive null. The `keyword`  is not changed into undefined because it was made clear to call the function only if keyword is not null : 
```
  const users = await (keyword
    ? usersService.getFilteredUsersStartingWith(keyword, userType, limit, nextToken)
    : usersService.getGroupedUsers(limit))
```